app/components/Form/ProjectEmissionIntensityReportFormSummary.tsx

```diff
@@ -0,0 +1,150 @@
+        isFirstRevision
+        summaryEmissionIntensityReportingRequirementFormChange: formChangesFor(
+          formDataTableName: "reporting_requirement"
+          reportType: "TEIMP"
+        ) {
+          edges {
+            node {
+              newFormData
+              operation
+              formChangeByPreviousFormChangeId {
+                newFormData
+              }
+            }
+          }
+        }
+        summaryEmissionIntensityReportFormChange: formChangesFor(
+          formDataTableName: "emission_intensity_report"
+        ) {
+          edges {
+            node {
+              newFormData
+              operation
+              formChangeByPreviousFormChangeId {
+                newFormData
+              }
+            }
+          }
+        }
+      }
+    `,
+    props.projectRevision
+  );
+
+  // Show diff if it is not the first revision and not view only (rendered from the overview page)
+  const renderDiff = !isFirstRevision && !props.viewOnly;
+
+  const summaryReportingRequirement =
+    summaryEmissionIntensityReportingRequirementFormChange?.edges[0]?.node;
+
+  const summaryEmissionIntensityReport =
+    summaryEmissionIntensityReportFormChange?.edges[0]?.node;
+
+  // Set the formSchema and formData based on showing the diff or not
+  const reportingRequirementDiffObject = !renderDiff
+    ? {
+        formSchema: emissionIntensityReportingRequirementSchema,
+        formData: summaryReportingRequirement?.newFormData,
+      }
+    : getFilteredSchema(
+        emissionIntensityReportingRequirementSchema as JSONSchema7,
+        summaryReportingRequirement?.newFormData || {}
+      );
+
+  const emissionIntensityReportDiffObject = !renderDiff
+    ? {
+        formSchema: emissionIntensityReportSchema,
+        formData: summaryEmissionIntensityReport?.newFormData,
+      }
+    : getFilteredSchema(
+        emissionIntensityReportSchema as JSONSchema7,
+        summaryEmissionIntensityReport?.newFormData || {}
+      );
+
+  // Set custom rjsf fields to display diffs
+  const customFields = { ...fields, ...CUSTOM_DIFF_FIELDS };
+
+  return (
+    <>
+      <h3>Emission Intensity Report</h3>
+      {/* Show this part if none of milestone report form properties have been updated */}
+      {Object.keys(reportingRequirementDiffObject.formSchema.properties)
+        .length === 0 &&
+        Object.keys(emissionIntensityReportDiffObject.formSchema.properties)
+          .length === 0 &&
+        summaryEmissionIntensityReport.operation !== "ARCHIVE" && (
+          <p>
+            <em>
+              Emission Intensity Report not{" "}
+              {isFirstRevision ? "added" : "updated"}
+            </em>
+          </p>
+        )}
+      {/* Show this part if the whole milestone report has been removed */}
+      {renderDiff && summaryEmissionIntensityReport.operation === "ARCHIVE" && (
+        <em className="diffOld">Emission Intensity Report Removed</em>
+      )}
```

Comment:
I don't think this needed because the only way to remove the whole report is to undo while first creating a project, which wouldn't show in the diffs